### ISSUE TITLE
Problem with settings

### ISSUE THREAD

**Issue by YoannFR** [acemod/ACE3#6551]
**Arma 3 Version:** `1.84` (stable / rc / dev)
**CBA Version:** `3.6.0` (stable / dev + commit hash)
**ACE3 Version:** `3.12.3` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
- I have this message when i spawn in editor : 
![af5dee5a72f8390f736b1aeebe918c9a](https://user-images.githubusercontent.com/27434438/44630469-7fbe5480-a95e-11e8-9b6b-f1c4ab2202d0.png)

And i have many messages in my rpt.

**Steps to reproduce:**
- Create mission and launch it.

**Where did the issue occur?**
- Dedicated / Editor (Singleplayer) / I don't have test for the others..

**Placed Modules:**
No module, it's the configuration in Addon Options

**RPT log file:**
https://pastebin.com/bbzHi7ji

**Comment by commy2** [comment]
RPT incomplete

**Comment by YoannFR** [comment]
https://pastebin.com/xbNqkfVL

But i search only to understand why i have this messages...

**Comment by commy2** [comment]
>22:30:09 [ACE] (common) TRACE: 19500 Reading missionConfigFile params: _paramsArray=[] z\ace\addons\common\functions\fnc_readSettingsFromParamsArray.sqf:23

@PabstMirror ?

**Comment by YoannFR** [comment]
I don't have the class Params ^^

**Comment by commy2** [comment]
The line comes from paramsArray variable being defined. Try on an empty mission, and the error should go away.

**Comment by dedmen** [comment]
>CBA Version: `3.6.0`

Your CBA is quite outdated.

**Comment by YoannFR** [comment]
I have updated my CBA. See my rpt : 
https://pastebin.com/QaYScGUv

**Comment by commy2** [comment]
Now try with CBA + ACE only instead of many mods in `@Agorapolis`.

**Comment by YoannFR** [comment]
https://pastebin.com/pzDCNCPm

With only CBA and ACE

**Comment by commy2** [comment]
>Xnooz_ACE_Extension_Anim.pbo
Neither CBA nor ACE

>kka3_ace_extension_anim.pbo
Neither CBA nor ACE

>10:02:03 Y:\Steam\steamapps\common\Arma 3\@ACE\addons\ace_backpacks.pbo - cd980dc2
>10:02:03 Y:\Steam\steamapps\common\Arma 3\@ACE\addons\ace_captives.pbo - unknown
Different commit hashes.

This Arma installation is a mess, because files are moved around inside the folders randomly.

**Comment by YoannFR** [comment]
The reason for the message is related to the configuration of ACE. I am trying to remove this message without modifying the mods.

**Comment by commy2** [comment]
No, I'm certain the reason for these messages is broken mods, which is why this should be tested with only CBA and ACE.

**Comment by YoannFR** [comment]
https://pastebin.com/FAqkjWQ4

What is your configuration of ACE ?

**Comment by commy2** [comment]
>15:50:43 Y:\Steam\steamapps\common\Arma 3\@ACE\addons\ace_backpacks.pbo - cd980dc2
>15:50:43 Y:\Steam\steamapps\common\Arma 3\@ACE\addons\ace_captives.pbo - unknown

Still two different commit hashes. And the CBA files are for some reason inside the `@ACE` folder.

I can upload a RPT next of my setup not before next Friday/Thursday evening, but the main difference is me not moving around PBO files into folders they do not belong.

**Comment by YoannFR** [comment]
I have two different commit hashes because i have modified ace_captives ( but it's the same problem with the original ) .

I have move the CBA files in @ACE for simplicity ;)

**Comment by commy2** [comment]
As far as I can see, nobody has this problem with the official builds.

**Comment by Monsen007** [comment]
> http://www.bilder-upload.eu/show.php?file=32fdd8-1535589041.png

> http://www.bilder-upload.eu/show.php?file=10089c-1535589330.png

`Global setting ace _vehiclelock_vehicleStartingLockState changed mid-missio. Mission restart may required to prevent issues`

and the old error message before the latest patch

> https://pastebin.com/cKDa7TMU


Have the same problem ...
had to first another error message after the patch was the way but for a new have now what synonymous of arma turned on by launcher and zack the next message.

before everything was okey with us in the clan as far as the mission was concerned where the modules were still to place and now we have the problem with unterander the if we save and joins new and continue the only me so the host what can take from boxes the others get hardly anything out times it's just ammo times it's only medic stuff but both is not anymore.

I'm my Latin at the end do not know what it is, have only cba_a3 in it and ace3 and the same

maybe one can help me ...

and sry google translator xD

write again in german :)

and I'm new here xD do not know me the equivalent xD

--------------------------------------------------------------------------------

Habe das gleiche problem ...
hatte zu erst eine andere fehler meldung nach den patch  war die weg aber dafür  eine neue hab jetzt auch was von arma angemacht per launcher und zack gleich die nächste meldung .

vorher war alles okey bei uns im clan was die mission angeht  wo die module noch so zu platzieren waren und jetzt haben wir das problem mit unterandern das wenn wir speichern und neu joinen und fortfahren  das nur ich also der host was aus kisten nehmen kann die andern bekommen kaum was raus mal sind es nur ammo mal is es nur medic zeug aber beides ist nicht mehr .

bin mir mein latein am ende weiss nicht woran es liegt  ,hab nur cba_a3 drin und ace3 und das selbe 

viellicht kann mir einer helfen ...

und sry google übersetzer xD

schreibe aber nochmal in deutsch :) 

und bin neu hier xD kenn mir dem entsprechen nicht raus xD

**Comment by commy2** [comment]
Completely different problem, as you don't have the message in the main menu background mission, but only after loading a savegame. Not related, so create your own issue report.

**Comment by Monsen007** [comment]
okey
